tests/test_rest.py

```diff
@@ -19,6 +19,8 @@
 import bottle
 import mock
 import six
+import voluptuous as volup
+import webtest
```

Comment:
`webtest` should be in the middle section. But there's no reason to touch this file, so I'll revert it.